### PR TITLE
@damassi => Some more artist tests

### PR DESCRIPTION
**Pull request by mzikherman**
- Related genes
- Current event
- Currently non-working one for filter (pended for now)

### CODE REVIEW

File: src/Apps/Artist/Routes/Overview/__test__/ArtworkFilter.test.tsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  //     query: graphql`
+  //       query ArtworkFilter_Test_Query {
+  //         artist(id: "pablo-picasso") {
+  //           ...ArtworkFilter_artist
+  //         }
+  //       }
+  //     `,
+  //     mockResolvers: { Artist: () => {} },
+  //   })
+  // }
```
Comment (damassi) [inline src/Apps/Artist/Routes/Overview/__test__/ArtworkFilter.test.tsx]: Can you add a `// FIXME: Uncomment tests, currently WIP` or something at the top of the file?

Comment (mzikherman) [reply src/Apps/Artist/Routes/Overview/__test__/ArtworkFilter.test.tsx]: Yea, will add, and clean up a bit. Wanted to PR this so I can possibly pair, or merge as a pended spec in the meantime, etc.

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit 33b252fa3580979581df00f162dcf6f00c6c2faf

https://deploy-preview-1443--artsy-reaction.netlify.com

File: src/Apps/Artist/Routes/Overview/__test__/ArtworkFilter.test.tsx (hunk 1 above)
Comment (damassi) [reply src/Apps/Artist/Routes/Overview/__test__/ArtworkFilter.test.tsx]: As long as these WIP sections are clearly labeled i think its ok, but better to just stash changes in a branch until ready

**Comment by alloy** [comment]
Alright, I’ve pushed some changes to your branch that make the right errors show up, which then revealed the real underlying error:

```
Error: Uncaught [Error: RelayMockNetworkLayerError: No mock defined for type "FormattedNumber"]
```

This error is about `FormattedNumber `, which is a custom scalar that MP has. Alas, custom scalars are not introspect-able, and so you have to redefine it wherever you want to use it 😞 (in general I’m not a huge fan of this polymorphic scalar, but that’s for another time).

After altering our mocked schema to include the definition for the scalar, the remaining error seems an expected one, namely the. fixture not stubbing all the expected data:

```
Failed: "Error: Error: Uncaught [TypeError: Cannot read property 'counts' of undefined]
```

Let me know if you’re able to continue work from here on or not.

**Comment by mzikherman** [comment]
💥 thank you!! Will check it out, that's a more expected error message coming from `render()`, so I should be able to take it from here.

**Comment by alloy** [comment]
@mzikherman Pushed one more commit to switch to the already updated version of `relay-mock-network-layer`.

**Comment by mzikherman** [comment]
Ok, merging this with working tests!

**Pull request merged**